ports/esp32/boards/ESP32_GENERIC_C2/sdkconfig.flash_2MiB

```diff
@@ -0,0 +1,17 @@
+# this variant has 2MiB flash
```

Comment:
As these settings are also valid for `sdkconfig.d2wd`, what do you think about making this file `ports/esp32/boards/sdkconfig.flash_2MiB` and including it for both this C2 variant and the D2WD variant.

(Will still need `sdkconfig.d2wd` for the two settings which are only needed on that board, for esptool).

Reply:
Good suggestion, will do it ...

App size is 1368192 bytes